src/PageContent/Settings/Settings.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import './StudentCoordinatorSettings.css';
+import OrganizationDetails from './OrganizationDetails';
+import AccountManager from './AccountManager';
+import './SCSettings0.css';
+import './SCSettings1.css';
+
+class Settings extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            account: null,
+            org: null,
+            coordinator: null,
+            secondaryContact: null,
+            isEditingOrg: false,
+            isEditingAcc: false,
+            email: 'phisigmaro@uw.edu'
+        };
+    }
+
+    // Backend TODO: Get data from DB
+    componentDidMount() {
+        let account  = {
+            user_type: 'admin',
+            name: 'Phi Sigma Ro',
+            email: this.state.email,
+            password: 'password',
+            organizationPhone: '209-992-9292',
+            address: {
+                street1: '4138 Brooklyn Ave NE',
+                street2: '',
+                city: 'Seattle',
+                state: 'WA',
+                zipcode: 98105
+            },
+            isVerified: true,
+            isActivated: true,
+            numVolunteers: 50,
+            coordinator: {
+                name: 'Andy Duncan',
+                email: 'andyd@uw.edu',
+                phone: '206-487-2859',
+                position: 'President'
+            },
+            secondaryContact: {
+                name: 'Jennifer Marly',
+                email: 'jmarly@uw.edu',
+                phone: '206-487-2992',
+                position: 'Vice President'
+            },
+            notifications: [
+                {
+                    type: 'recurring_pickup_request',  // Enums.NotificationType
+                    content: '-L5QoXeC_UrL5tRRED3e'  // key of DeliveryRequest
+                }
+            ],
+            smsNotif: true,
+            emailNotif: true
+        };
+        this.setState({
```

Comment:
same, don't need to set any of these, use `props.account`

Reply:
Fixed.